### ISSUE TITLE
Classification accuracy on validation set didn't improve while fine-tuning BERT

### ISSUE THREAD

**Issue by rxlian** [huggingface/transformers#5648]
# ❓ Questions & Help

I was fine-tuning BERT with BertForSequenceClassification on my own dataset. 0.8 of the whole dataset was used as training set and the others as validation set. The training loss decreased during the process. However, the accuracy on validation set was always around 0.5, which is similar to random guessing. And the accuracy on validation didn't improve a lot after fine-tuning. For example, from epochs 1-3, accuracy was from 0.48-0.52. So I was wondering whether this problem was caused by my dataset itself or if I did something wrong while fine-tuning it? Does anybody have any ideas on this? By the way, before this, I was fine-tuning BERT on another dataset and it did improve the classification accuracy a lot .

**A link to original question on Stack Overflow**:

**Comment by LysandreJik** [comment]
Hello! This is an interesting question, but is kind of out of scope for the Github issues. We just opened a forum at [discuss.huggingface.co](https://discuss.huggingface.co). Do you think you could ask your question over there?

Thank you.
